Makefile

```diff
@@ -387,7 +387,12 @@ istioctl-install:
 .PHONY: test localTestEnv test-bins
 
 # Run coverage tests
-test: pilot-test mixer-test security-test broker-test galley-test common-test
+JUNIT_UNIT_TEST_XML ?= $(ISTIO_OUT)/junit_unit_tests.xml
+test: | $(ISTIO_OUT)
```

Comment:
You don't need to (and probably shouldn't) add the "| $(ISTIO_OUT)" since you've added the mkdir.  The "| $(ISTIO_OUT)" basically just causes make to create the directory if it doesn't already exist.